metagpt/provider/openai_api.py

```diff
@@ -323,3 +325,29 @@ async def amoderation(self, content: Union[str, list[str]]):
     async def _amoderation(self, content: Union[str, list[str]]):
         rsp = await self.llm.Moderation.acreate(input=content)
         return rsp
+
+    @staticmethod
```

Comment:
why staticmethod?

Reply:
> why staticmethod?

Because it does not use any members of self.